playbooks/configure-apt-sources.yml

```diff
@@ -126,6 +146,20 @@
       when:
         - apt_artifact_found | bool
         - apt_artifact_enabled | bool
+        - apt_artifact_mode == "strict"
+
+    # Set rpco_apt_sources_restore_source to 'local' to have the
+    # original apt sources file from the deploy host be pushed out
+    # to all hosts and containers.
+    - name: Restore the original sources file (artifact 'loose' mode)
+      copy:
+        src: /etc/apt/sources.list.original
+        dest: /etc/apt/sources.list
+        remote_src: "{{ rpco_apt_sources_restore_source | default('remote') == 'remote' }}"
+        force: yes
+      register: apt_sources_restore
+      when:
```

Comment:
This should do nothing if artifacting are disabled.

Reply:
Good catch - updating.